src/Module/Update/Channel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2023, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Update;
+
+use Friendica\Content\Conversation;
+use Friendica\Core\System;
+use Friendica\DI;
+use Friendica\Module\Conversation\Channel as ChannelModule;
+
+/**
+ * Asynchronous update module for the Channel page
+ *
+ * @package Friendica\Module\Update
+ */
+class Channel extends ChannelModule
+{
+	protected function rawContent(array $request = [])
+	{
+		$this->parseRequest($request);
+
+		$o = '';
+		if (!empty($request['force'])) {
+			$o = DI::conversation()->render(self::getItems($request), Conversation::MODE_CHANNEL, true, false, 'created', DI::userSession()->getLocalUserId());
```

Comment:
Now that `getItems` is an object method, it should be called with `$this`. You can also reuse `$this->conversation` and `$this->session` but you have to make them `protected` instead of `private`.
```suggestion
			$o = $this->conversation->render($this->getItems($request), Conversation::MODE_CHANNEL, true, false, 'created', $this->session->getLocalUserId());
```

Reply:
Changed